backend/api_v2/deployment_helper.py

```
@@ -248,3 +252,169 @@ def get_execution_status(execution_id: str) -> ExecutionResponse:
+        Args:
+            url (str): The presigned URL to fetch the file from
+
+        Returns:
+            InMemoryUploadedFile: The fetched file as an uploaded file object
+
+        Raises:
+            PresignedURLFetchError: If the file cannot be fetched
+        """
+        parsed_url = urlparse(url)
+        sanitized_url = parsed_url._replace(query="").geturl()  # For logging
+        host = (parsed_url.hostname or "").lower()
```

Ruff (0.12.2)

Local variable `host` is assigned to but never used

Remove assignment to unused variable `host`

(F841)